ditatodocbook.sh

```diff
@@ -85,6 +90,7 @@ CLEANTEMP=1
 CLEANID=0
 TWEAK=""
 ENTITYFILE="entities.xml"
+MAP2MAIN_XSLT="$mydir/map-to-MAIN.xsl"
```

Comment:
What is this good for? Especially among the "configurable" options, this seems misplaced.

Reply:
This is only used once. And it definitely should not be grouped together with the user-configurable options.